reconnecting-eventsource.d.ts

```diff
@@ -0,0 +1,21 @@
+export interface Configuration extends EventSourceInit {
```

Comment:
How about calling this `ReconnectableEventSourceInit` instead of `Configuration`?